src/org/traccar/protocol/Jt600ProtocolDecoder.java

```diff
@@ -256,11 +259,29 @@ private Position decodeU01(String sentence, Channel channel, SocketAddress remot
         position.set(Position.KEY_CID, parser.nextInt());
         position.set(Position.KEY_LAC, parser.nextInt());
         position.set(Position.KEY_GSM, parser.nextInt());
-        position.set(Position.KEY_ODOMETER, parser.nextLong() * 1000);
+        position.set(Position.KEY_ODOMETER, parser.nextLong() * 100);
```

Comment:
Yes, I am pretty sure. Here is what documentation says:

> | NO. | Name | Value(Extended ASCII) | Description |
> | --- | --- | --- | --- | --- |
> | 19 |  Mileage | 100 | The current value of the mileage of 100 kilometers |

And it does matches with what is coming from device. Also I believe it's weird to count mileage by thousands of kilometers.